Header header

uint8 STATE_DIRECT_CTRL = 0
uint8 STATE_JS_CTRL = 1
uint8 STATE_JS_NO_CTRL = 2
uint8 STATE_JS_RESET_POS_CTRL = 3
uint8 STATE_CMD_HOVER = 4
uint8 STATE_CMD_CTRL = 5
uint8 STATE_CMD_NO_CTRL = 6
uint8 STATE_CMD_RESET_POS_CTRL = 7
uint8 state

uint32 last_traj_id